sdk/formrecognizer/azure-ai-formrecognizer/tests/recordings/test_dac_analyze_prebuilts_from_url.pyTestDACAnalyzePrebuiltsFromUrltest_business_card_multipage_pdf.json

```diff
@@ -9,49 +9,51 @@
         "Connection": "keep-alive",
```

Comment:
The recordings are added due to the test class rename and some other smaller renames that affected recordings